### DISCUSSION TITLE
Project manager overview dasboard

### DISCUSSION THREAD

**Discussion by bluebrad** [Leantime/leantime#1993] [Ideas]
I would like to see a dashboard and new leveling system for project managers. They would have oversight over all tasks and could redelegate as needed. This would also provide them with real-time activity feeds on active users. this way in the daily stand up there could also be a task for checks on projects.

**Comment by gloriafolaron** [comment]
I'm having a hard time imagining the workflow on this. Is there a way you could help me visualize it a bit better?

**Reply by FelixAddison** [reply]
I also think this would be very useful (essential really), especially in cases where tasks in one project affect progress in another or shared resources are being used.

A dynamic mindmap or diagram view that would show both task and project inter-relationships could be ideal for this.


https://www.yworks.com/ has been doing some really amazing work on data visualization with diagrams and their software is mostly free to use.
Perhaps some sort of integration would allow this type of visualization?

**Comment by bluebrad** [comment]
ill see about building a user interface demo
This would be a dashboard design for Project managers so they can see who
is where and all projects they created.
also be able to see all tasks unlike the task window for there own and
filters would allow for by project, sprint, milestone, person.

**Reply by gloriafolaron** [reply]
So the second part -- all tasks across all projects will likely end up being a view on our program  plan management. I'm wondering if the rest may fit there as well from the way I think I'm understanding it? 

A wireframe would be helpful to imagine too :) 

Thanks for the ideas.

**Comment by bluebrad** [comment]
still drafting it up.
